discovery-infra/tests/base_test.py

```diff
@@ -345,3 +348,27 @@ def kube_api_context(self, kube_api_client):
 
         with kube_api_context:
             yield kube_api_context
+
+    @pytest.fixture(scope="function")
+    def update_olm_config(self) -> Callable:
```

Comment:
I you remove the return value (few comments next) change the return type annotation to 
```suggestion
    def update_olm_config(self) -> Callable[[TerraformConfig, ClusterConfig, <TypeOfOperators>], None]:
```
The `None` part is the return value type

Reply:
OK

Reply:
It is not working due to `TypeError: 'ABCMeta' object is not subscriptable`